pkg/api/types.go

```diff
@@ -2103,6 +2103,12 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined in
+	// Admission Controller config or pod will be rejected.
+	// If not specified, the pod priority will be zero.
```

Comment:
I updated the proposal a few minutes ago and added more details about the default name resolution. We probably go with resolving unspecified PriorityClassName to zero in the initial version, but in a follow up change, we will add the support to annotate one of the PriorityClasses with a "default" annotation. Once such a priority class exists, the system resolves unspecified PriorityClassName to the value of the default class. If there is no such a class, we will still resolve to zero. Zero is not the lowest possible priority in the system given that any valid integer is a valid priority.